ui/admin/app/components/model-providers/ModelProviderForm.tsx

```diff
@@ -119,52 +145,79 @@ export function ModelProviderForm({
+                    >
+                        {requiredConfigParamFields.fields.map((field, i) => (
+                            <ControlledInput
+                                key={field.id}
+                                label={field.label}
+                                control={form.control}
+                                name={`requiredConfigParams.${i}.value`}
+                                classNames={{
+                                    wrapper: "flex-auto bg-background",
+                                }}
+                            />
+                        ))}
+                    </form>
+                </Form>
+
+                {showCustomConfiguration ? (
+                    <>
+                        <Separator className="my-4" />
+
+                        <div className="flex items-center gap-2">
+                            <TypographyH4 className="font-semibold text-md">
+                                Custom Configuration (Optional)
+                            </TypographyH4>
+                            {ModelProviderConfigurationLinks[
+                                modelProvider.id
+                            ] ? (
+                                <Button variant="ghost" size="icon">
+                                    <Link
+                                        to={
+                                            ModelProviderConfigurationLinks[
+                                                modelProvider.id
+                                            ]
+                                        }
+                                    >
+                                        <CircleHelpIcon className="text-muted-foreground" />
+                                    </Link>
+                                </Button>
+                            ) : null}
+                        </div>
+                        <NameDescriptionForm
+                            defaultValues={form.watch(
+                                "additionalConfirmParams"
+                            )}
+                            onChange={(values) =>
+                                form.setValue("additionalConfirmParams", values)
+                            }
                         />
-                    ))}
-                </form>
-            </Form>
-
-            <Separator className="my-4" />
-
-            <TypographyH4 className="font-semibold text-md">
-                Custom Configuration (Optional)
-            </TypographyH4>
-            <NameDescriptionForm
-                defaultValues={form.watch("additionalConfirmParams")}
-                onChange={(values) =>
-                    form.setValue("additionalConfirmParams", values)
-                }
-            />
+                    </>
+                ) : null}
+            </div>
 
-            <div className="flex justify-end">
+            <div className="flex justify-end px-6 py-4 border-t border-t-secondary">
```

Comment:
2 things here:
1. Darren and I have agreed that we should actively try to only use borders where absolutely necessary. We can probably remove them from dialog headers/footers

Reply:
Got it, thanks! Will axe this and I think there's another one, will axe that as well!